wd_helper.au3

```diff
@@ -338,7 +339,8 @@ EndFunc   ;==>_WD_LinkClickByText
 ;                  |$_WD_OPTION_None    (0) = No optional feature processing
 ;                  |$_WD_OPTION_Visible (1) = Confirm element is visible
 ;                  |$_WD_OPTION_Enabled (2) = Confirm element is enabled
-;                  |$_WD_OPTION_NoMatch (8) = Confirm element not found
+;                  |$_WD_OPTION_NoMatch (8) = Confirm element is not found
+;                  |$_WD_OPTION_Hidden (16) = Confirm element is not visible
```

Comment:
Please also add the adequate feature for DISABLED

Reply:
I understand why you would request that. However, there are several reasons why I won't be adding it now --

- It's outside the scope of this PR
- I'm not sure that it is needed
- It would complicate the logic associated with $iOptions. 

If you would like to discuss further, please open a new issue where this dialog can continue.